.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "node"
+  - "6"
+cache: yarn
+before_install:
```

Comment:
The script is necessary, it is installing the RC version of yarn 0.28.4, yarn 0.27.x has an issue with symlinking the correct version .bin file sometimes - https://github.com/yarnpkg/yarn/issues/3801,